log.py

```diff
@@ -24,11 +24,11 @@ def log_debbug(requests, debbug_path, debbug_name, mode):
     data = []
     if mode == "w":
         data.append(["id", "timestamp", "created_time", "latency",
```

Comment:
Why are we still needing the done column? Status already tells us this information.

Reply:
What if both logs are wrong? What you're saying is that you have a log message to confirm that another log message is correct? To me, it looks like a receipt for confusion.

If you would like to make sure that something is correct, test it. Having two log messages saying the same thing does look like the way to go.

Reply:
I see. Your argument makes sense to me. Going to remove it in next commit.